src/utils/audio-manager.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+const playMock = jest.fn();
+Audio.prototype.play = async () => playMock();
```

Comment:
Will it work if you just use:

```js
Audio.prototype.play = jest.fn();
```
?

Reply:
it doesn't work, there we have a problem in line 12 audio-manager.js while catching the error:  TypeError: Cannot read properties of undefined (reading 'catch')